rzup/Cargo.toml

```diff
@@ -12,15 +12,29 @@ name = "rzup"
 path = "src/bin/rzup.rs"
 
 [features]
-default = ["cli", "install"]
+default = ["cli", "install", "publish"]
```

Comment:
I'm thinking maybe this shouldn't be the default so that it's easier for folks to test out an internal version.

Reply:
We could easily do this, I'm not sure I understand the reasoning though. Just compiling in publishing support shouldn't really affect must of anything. It should be a small amount of extra compilation time and code.